app/lib/clients/bambooHR_client.rb

```diff
@@ -0,0 +1,45 @@
+require 'bamboozled'
+require 'date'
+
+
+class Clients::BambooHRClient
+    client = Bamboozled.client(subdomain: AppEnv['BAMBOOHR_SUBDOMAIN'], api_key: AppEnv['BAMBOOHR_API_KEY'])
+    # birthday = client.employee.all("birthday")
+    # anniversary = client.employee.all("hireDate")
+    day_before_yesterday = (Date.today - 2).to_formatted_s(:iso8601)
+    yesterday = (Date.today - 1).to_formatted_s(:iso8601)
+    today = (Date.today).to_formatted_s(:iso8601)
+    tomorrow = (Date.today + 1).to_formatted_s(:iso8601)
+    day_afer_tomorrow = (Date.today + 2).to_formatted_s(:iso8601)
+    time_off = client.time_off.whos_out(Time.now, today)
+
+    # puts birthday
+    # puts anniversary
+    puts "time off: "+time_off.to_s
+
+
+    employees_info = client.report.custom(["id", "birthday", "hireDate", "isPhotoUploaded"], format = "JSON")
+    employees = []
+
+    for employee in employees_info
+        #and employee['birthday'] != "" included because 2 of the employees in the test company do not have birthdays listed
+        if day_before_yesterday.include? employee['birthday'] or yesterday.include? employee['birthday'] or today.include? employee['birthday'] or tomorrow.include? employee['birthday'] or day_afer_tomorrow.include? employee['birthday'] and employee['birthday'] != ""
```

Comment:
Things like this could made a little more functional with Enumerable methods:
```
if employee['birthday'].present? &&
     [day_before_yesterday, yesterday, today, tomorrow].any? { |d| 
       d.include? employee['birthday']
     }
```
https://ruby-doc.org/core-2.7.2/Enumerable.html#method-i-any-3F